postgres-openssl/src/lib.rs

```diff
@@ -73,14 +73,17 @@ use tokio_postgres::tls::{ChannelBinding, TlsConnect};
 #[cfg(test)]
 mod test;
 
+type ConfigCallback =
+    dyn Fn(&mut ConnectConfiguration, &str) -> Result<(), ErrorStack> + Sync + Send;
+
```

Comment:
Clippy complains if we don't use a type alias here. To be fair, it is a pretty lengthy type declaration. 😅